demo/meeting/src/containers/LocalMediaStreamMetrics/index.tsx

```diff
@@ -0,0 +1,176 @@
+                    : '';
+                })
+              }
+            />
+            <MetricItem
+              metricName="Frame Rate"
+              metricValues={
+                ssrcArray &&
+                ssrcArray.map(ssrc => {
+                  return localVideoStreamMetrics[ssrc]
+                    .videoUpstreamFramesEncodedPerSecond
+                    ? localVideoStreamMetrics[
+                        ssrc
+                      ].videoUpstreamFramesEncodedPerSecond.toString()
+                    : '';
+                })
+              }
+            />
+            <MetricItem
+              metricName="Frame Height"
+              metricValues={
+                ssrcArray &&
+                ssrcArray.map(ssrc => {
+                  return localVideoStreamMetrics[ssrc].videoUpstreamFrameHeight
+                    ? localVideoStreamMetrics[
+                        ssrc
+                      ].videoUpstreamFrameHeight.toString()
+                    : '';
+                })
+              }
+            />
+            <MetricItem
+              metricName="Frame Width"
+              metricValues={
+                ssrcArray &&
+                ssrcArray.map(ssrc => {
+                  return localVideoStreamMetrics[ssrc].videoUpstreamFrameWidth
+                    ? localVideoStreamMetrics[
+                        ssrc
+                      ].videoUpstreamFrameWidth.toString()
+                    : '';
+                })
+              }
+            />
+            <MetricItem
+              metricName="Frame Height"
+              metricValues={
+                ssrcArray &&
+                ssrcArray.map(ssrc => {
+                  return localVideoStreamMetrics[ssrc]
+                    .videoUpstreamGoogFrameHeight
+                    ? localVideoStreamMetrics[
+                        ssrc
+                      ].videoUpstreamGoogFrameHeight.toString()
+                    : '';
+                })
+              }
+            />
+            <MetricItem
+              metricName="Frame Width"
+              metricValues={
+                ssrcArray &&
+                ssrcArray.map(ssrc => {
+                  return localVideoStreamMetrics[ssrc]
+                    .videoUpstreamGoogFrameWidth
+                    ? localVideoStreamMetrics[
+                        ssrc
+                      ].videoUpstreamGoogFrameWidth.toString()
+                    : '';
+                })
+              }
+            />
+          </MediaStatsList>
+        </>
+      )}
+      {hasBandwidthInfo && (
+        <>
+          <PopOverHeader title={'Bandwidth statistics'} />
+          <MediaStatsList>
+            <MetricItem metricName="" metricValues={['Outgoing', 'Incoming']} />
+            <MetricItem
+              metricName="Bandwidth (kbps)"
+              metricValues={[
+                availableOutgoingBandwidth !== null ? availableOutgoingBandwidth.toString() : "",
```

Comment:
`hasBandwidthInfo` checks this already right, so do we need the check again here?

Reply:
If I don't add check here, typescript will show error of : hasBandwidthInfo is possibly "nulll"

Reply:
Got it! Thanks for the explanation.